### ISSUE TITLE
Project Feedback

### ISSUE THREAD

**Issue by qassisju** [qassisju/MI-449-SS17-741-js-package-json-intro-AjvZZg#1]
@KatieMFritz Can you take a look at this? It's [hosted here](https://qassisju-kg-tgc7noax99ajvzzg.surge.sh/) and meets the following criteria:

- [x] Start your project with [these old game files](https://gist.github.com/egillespie/61d72035cd8c2639f07db1dbe2010605)
- [x] Convert the game's CSS to Sass, using variables, functions, and nesting where appropriate
- [x] Write a `dev` script in `package.json` that auto-compiles Sass and refreshes the browser at the same time
- [x] Write a `build` script in `package.json` that lints HTML, Sass, and JavaScript files before compiling Sass files (you can use these linters: [htmlhint](https://www.npmjs.com/package/htmlhint) for HTML, [stylelint](https://www.npmjs.com/package/stylelint) and [stylelint-config-standard](https://github.com/stylelint/stylelint-config-standard#installation) for Sass, [standard](https://www.npmjs.com/package/standard) for JavaScript)
- [x] All linting problems are fixed (problems will be displayed in the terminal when you run `yarn build`)
- [x] Write a `deploy` script in `package.json` that builds the project and uploads it to Surge if there are no errors
- [x] Unnecessary files like `node_modules` and your compiled CSS do not exist in your repository
- [x] Unnecessary files like `package.json` and `yarn.lock` are not uploaded to Surge

**Comment by egillespie** [comment]
Hi @qassisju, this is a pretty good start. There are a couple of things that stand out:

### Styles are not working

When I visit your hosted site, your page looks like this:

![image](https://cloud.githubusercontent.com/assets/5572255/25053803/8462f95c-2127-11e7-9f0a-0a0f130c107a.png)

None of the styles are applying. Can you please correct this?

### Linting is missing

It looks like you missed the fourth criteria for this project. Please introduce linters for HTML, JS, and Sass and make sure they all get called as part of your `build` script.

---

After you’ve made your changes and pushed them to GitHub and your hosted site, give it a once-over to make sure it looks right, then comment back here and I’ll take another look.

Thanks! 🏍

**Comment by qassisju** [comment]
I for sure fixed the style issue and I think I fixed the linting issues as well, though that one Im not as sure of

**Comment by egillespie** [comment]
Yep, the styles are fixed. The linting is still missing, though. You'll have to add three new scripts and four new dev dependencies (one for each of the linked packages). In order to add the linting for each one you will have to read the documentation about how to use the linters and/or find examples to follow.

**Comment by qassisju** [comment]
Hopefully the listing should be added successfully!

**Comment by egillespie** [comment]
This looks fantastic! My last recommendation is that you rename your lint scripts to start with `lint:` instead of `build:`. The reason for this is that `build` implies that new files will be created. Since linters don't create new files, they're not really building anything.

When you make this change, make sure to update your `build` script to call `lint:*` _before_ calling `build:*` so you lint/verify your files before you try to compile anything.

Let me know when you've made the change and want me to take another look! 🏆

**Comment by qassisju** [comment]
Okay, I gave it a shot

**Comment by egillespie** [comment]
Looking good. You should update your `deploy` script to run `build` so that it will lint and build before deploying. Right now it only builds and then deploys.

**Comment by qassisju** [comment]
I think you meant to say add lint to the deploy script? Check to see if this is done correctly

**Comment by egillespie** [comment]
Oh, yeah. You're on the right track. Linting should happen _before_ building though. 😉

**Comment by qassisju** [comment]
Okay it should be that way now

**Comment by egillespie** [comment]
Yep, looks good! :shipit:
